public/app/features/alerting/unified/mocks.ts

```diff
@@ -593,6 +619,18 @@ export const mockFolder = (partial?: Partial<FolderDTO>): FolderDTO => {
   };
 };
 
+export const mockUser = (partial?: Partial<UserDTO>): UserDTO => {
```

Comment:
Maybe we can re-use `getMockUser()`? 

https://github.com/grafana/grafana/blob/f7c6d9b227582937dcf2ca4f31eabba748bf05bd/public/app/features/users/__mocks__/userMocks.ts#L38-L50

Reply:
Thanks, using this now